### PR TITLE
Rinto/ni 109 create docs internet gateway

### PR DESCRIPTION
**Pull request by rintooo**
Added docs of Internet Gateway / Gateway Interface / Public IP / Static Route / Image / ImageMember / ImageMemberAccepter resources.

### CODE REVIEW

File: website/docs/d/imagestorages_image_v2.html.markdown [hunk 1]
```diff
@@ -0,0 +1,66 @@
+---
+layout: "ecl"
+page_title: "Enterprise Cloud: ecl_imagestorages_image_v2"
+sidebar_current: "docs-ecl-datasource-imagestorages-image-v2"
+description: |-
+  Get information on an Enterprise Cloud Image.
+---
+
+# ecl\_network\_internet_gateway\_v2
```
Comment (keiichi-hikita) [inline website/docs/d/imagestorages_image_v2.html.markdown]: typo "internet gateway"

File: website/docs/d/network_gateway_interface_v2.html.markdown [hunk 2]
```diff
@@ -0,0 +1,93 @@
+---
+layout: "ecl"
+page_title: "Enterprise Cloud: ecl_network_gateway_interface_v2"
+sidebar_current: "docs-ecl-datasource-network-gateway_interface-v2"
+description: |-
+  Get information on an Enterprise Cloud Gateway interface.
+---
+
+# ecl\_network\_gateway_interface\_v2
```
Comment (keiichi-hikita) [inline website/docs/d/network_gateway_interface_v2.html.markdown]: add backslash before "_" between "gateway" and "interface"

File: website/docs/d/network_gateway_interface_v2.html.markdown [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
+* `azure_gw_id` - (Optional) Azure Gateway to which this port is connected.
+
+* `description` - (Optional) Description of the Gateway Interface resource.
+
+* `gateway_interface_id` - (Optional) Unique ID of the Gateway Interface resource.
+
+* `gcp_gw_id` - (Optional) GCP Gateway to which this port is connected.
+
+* `gw_vip_v4` - (Optional) IP version 4 address to be assigned virtual router on VRRP.
```
Comment (keiichi-hikita) [inline website/docs/d/network_gateway_interface_v2.html.markdown]: schema name is "gw_vipv4" . Unmatch bw doc and actual schema.
ipv6 has same problem.

File: website/docs/d/network_gateway_interface_v2.html.markdown [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+
+* `primary_ipv6` - (Optional) IP version 6 address to be assigned to primary device on VRRP.
+
+* `secondary_ipv4` - (Optional) IP version 4 address to be assigned to secondary device on VRRP.
+
+* `secondary_ipv6` - (Optional) IP version 6 address to be assigned to secondary device on VRRP.
+
+* `service_type` - (Optional) Service type for this interface. Must be one of "aws", "azure", "gcp", "interdc", "internet" and "vpn".
+
+* `status` - (Optional) The Gateway Interface status..
```
Comment (keiichi-hikita) [inline website/docs/d/network_gateway_interface_v2.html.markdown]: There is duplicate "."

File: website/docs/d/network_gateway_interface_v2.html.markdown (hunk 3 above)
Comment (keiichi-hikita) [reply website/docs/d/network_gateway_interface_v2.html.markdown]: There are same problems in attributes section.

File: website/docs/d/network_internet_gateway_v2.html.markdown [hunk 5]
```diff
@@ -0,0 +1,52 @@
+---
+layout: "ecl"
+page_title: "Enterprise Cloud: ecl_network_internet_gateway_v2"
+sidebar_current: "docs-ecl-datasource-network-internet_gateway-v2"
+description: |-
+  Get information on an Enterprise Cloud Internet gateway.
+---
+
+# ecl\_network\_internet_gateway\_v2
```
Comment (keiichi-hikita) [inline website/docs/d/network_internet_gateway_v2.html.markdown]: "\" problem.

File: website/docs/d/network_internet_gateway_v2.html.markdown [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+
+* `internet_gateway_id` - (Optional) Unique ID of the Internet Gateway resource.
+
+* `internet_service_id` - (Optional) Internet Service instantiated by this Gateway.
+
+* `name` - (Optional) Name of the Internet Gateway resource.
+
+* `qos_option_id` - (Optional) Quality of Service options selected for this Gateway.
+
+* `status` - (Optional) The Internet Gateway status..
```
Comment (keiichi-hikita) [inline website/docs/d/network_internet_gateway_v2.html.markdown]: dup "."

File: website/docs/d/network_public_ip_v2.html.markdown [hunk 7]
```diff
@@ -0,0 +1,54 @@
+---
+layout: "ecl"
+page_title: "Enterprise Cloud: ecl_network_public_ip_v2"
+sidebar_current: "docs-ecl-datasource-network-public_ip-v2"
+description: |-
+  Get information on an Enterprise Cloud Public ip.
+---
+
+# ecl\_network\_public_ip\_v2
```
Comment (keiichi-hikita) [inline website/docs/d/network_public_ip_v2.html.markdown]: lack of "\"

File: website/docs/d/network_public_ip_v2.html.markdown [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+* `submask_length` - (Optional) Specifies the size of the block by the length of its subnetwork mask length.
+
+* `tenant_id` - (Optional) Tenant ID of the owner (UUID).
+
+
+## Attributes Reference
+
+The following attributes are exported:
+`id` is set to the ID of the found public ip. In addition, the following attributes are exported:
+* `region` - See Argument Reference above.
```
Comment (keiichi-hikita) [inline website/docs/d/network_public_ip_v2.html.markdown]: Is it okay that there is no blank line between row "`id` is set ... " and the row "region` ..." .

File: website/docs/d/network_static_route_v2.html.markdown [hunk 9]
```diff
@@ -0,0 +1,71 @@
+---
+layout: "ecl"
+page_title: "Enterprise Cloud: ecl_network_static_route_v2"
+sidebar_current: "docs-ecl-datasource-network-static_route-v2"
+description: |-
+  Get information on an Enterprise Cloud Static route.
+---
+
+# ecl\_network\_static_route\_v2
```
Comment (keiichi-hikita) [inline website/docs/d/network_static_route_v2.html.markdown]: "\", static and route

File: website/docs/r/imagestorages_image_v2.html.markdown [hunk 10]
```diff
@@ -0,0 +1,87 @@
...
+### Basic Image
+
+```hcl
+resource "ecl_imagestorages_image_v2" "image_1" {
+    name  = "Temp_Terraform_AccTest"
+    local_file_path = "/tmp/tempfile.img"
+    container_format = "bare"
+    disk_format = "qcow2"
+
+    timeouts {
```
Comment (keiichi-hikita) [inline website/docs/r/imagestorages_image_v2.html.markdown]: I think we'd better to remove timeout section from basic example.
This parameter won't be used in basic case.

File: website/docs/r/imagestorages_image_v2.html.markdown [hunk 11]
```diff
@@ -0,0 +1,87 @@
+---
+layout: "ecl"
+page_title: "Enterprise Cloud: ecl_imagestorages_image_v2"
+sidebar_current: "docs-ecl-resource-imagestorages-image-v2"
+description: |-
+  Manages a V2 image resource within Enterprise Cloud.
+---
+
+# ecl\_imagestorages\_image_v2
```
Comment (keiichi-hikita) [inline website/docs/r/imagestorages_image_v2.html.markdown]: "\" before v2.

File: website/docs/r/imagestorages_image_v2.html.markdown [hunk 12]
```diff
@@ -0,0 +1,87 @@
...
+* `visibility` - (Optional) Scope of image accessibility. Must be one of "public", "private". Defaults to "private".
+
+* `peroperties` - (Optional) A map of key/value pairs to set freeform information about an image.
+
+
+## Attributes Reference
+
+The following attributes are exported:
+
+* `region` - See Argument Reference above.
```
Comment (keiichi-hikita) [inline website/docs/r/imagestorages_image_v2.html.markdown]: It looks there are some lack of attribute.
Parameter in argument should be appeared at attribute, right?

File: website/docs/r/imagestorages_member_accepter_v2.html.markdown [hunk 13]
```diff
@@ -0,0 +1,49 @@
+---
+layout: "ecl"
+page_title: "Enterprise Cloud: ecl_imagestorages_member_accepter_v2"
+sidebar_current: "docs-ecl-resource-imagestorages-member_accepter-v2"
+description: |-
+  Manages a V2 Image member resource within Enterprise Cloud.
+---
+
+# ecl\_imagestorages\_member_accepter_v2
```
Comment (keiichi-hikita) [inline website/docs/r/imagestorages_member_accepter_v2.html.markdown]: lack of "\"

File: website/docs/r/imagestorages_member_v2.html.markdown [hunk 14]
```diff
@@ -0,0 +1,50 @@
+---
+layout: "ecl"
+page_title: "Enterprise Cloud: ecl_imagestorages_member_v2"
+sidebar_current: "docs-ecl-resource-imagestorages-member-v2"
+description: |-
+  Manages a V2 Image member resource within Enterprise Cloud.
+---
+
+# ecl\_imagestorages\_member_v2
```
Comment (keiichi-hikita) [inline website/docs/r/imagestorages_member_v2.html.markdown]: "\" before v2

File: website/docs/r/imagestorages_member_v2.html.markdown [hunk 15]
```diff
@@ -0,0 +1,50 @@
...
+
+The following attributes are exported:
+
+* `region` - See Argument Reference above.
+* `created_at` - Date and time of image member creation.
+* `schema` - URL for schema of the member.
+* `status` - The status of this image member.
+* `updated_at` - Date and time of last modification of image member.
+
+## Import
```
Comment (keiichi-hikita) [inline website/docs/r/imagestorages_member_v2.html.markdown]: If import is not possible, we can remove this section itself.

File: website/docs/r/network_gateway_interface_v2.html.markdown [hunk 16]
```diff
@@ -0,0 +1,104 @@
+---
+layout: "ecl"
+page_title: "Enterprise Cloud: ecl_network_gateway_interfacev2"
+sidebar_current: "docs-ecl-resource-network-gateway_interface-v2"
+description: |-
+  Manages a V2 gateway interface resource within Enterprise Cloud.
+---
+
+# ecl\_network\_gateway_interface_v2
```
Comment (keiichi-hikita) [inline website/docs/r/network_gateway_interface_v2.html.markdown]: lack of "\"

File: website/docs/r/imagestorages_member_accepter_v2.html.markdown [hunk 17]
```diff
@@ -0,0 +1,49 @@
...
+
+
+## Example Usage
+
+### Basic Image
+
+```hcl
+resource "ecl_imagestorages_member_accepter_v2" "accepter_1" {
+	provider = "ecl.accepter_tenant"
+	image_member_id = "${ecl_imagestorages_member_v2.member_1.id}"
```
Comment (keiichi-hikita) [inline website/docs/r/imagestorages_member_accepter_v2.html.markdown]: I think it is not better to use reference to other resource by using "$(variable)".

Comment (rintooo) [reply website/docs/r/imagestorages_member_accepter_v2.html.markdown]: Thx for the comment.
Will keep variables, but also write referred resource "member".

**Pull request merged**